packages/cloudflare/src/cli/build/patches/to-investigate/inline-eval-manifest.ts

```diff
@@ -13,7 +13,9 @@ export function inlineEvalManifest(code: string, config: Config): string {
   console.log("# inlineEvalManifest");
   const manifestJss = globSync(
     path.join(config.paths.standaloneAppDotNext, "**", "*_client-reference-manifest.js").replaceAll("\\", "/")
-  ).map((file) => file.replaceAll("\\", "/").replace(`${config.paths.standaloneApp.replaceAll("\\", "/")}/`, ""));
+  ).map((file) =>
```

Comment:
maybe a cleaner way to fix this would be to 
- use `.replaceAll(path.sep, path.posix.sep)`
- move that in an helper function
- apply that everywhere we have path, i.e. here and where `config.paths...` are set

Reply:
> apply that everywhere we have path, i.e. here and where `config.paths...` are set

I think there may be side effects if we apply this everywhere

It's just something wrong with `glob`

Reply:
I have now replaced with `.replaceAll(path.sep, path.posix.sep)`

I think `value.replace...` is better than `fixPath(value)` because we don't need to wrap the `value` here in `()`, and if we want to create an extension method we need to use `String.prototype...`

Reply:
I think extracting the (...).replaceAll(..., ...) to a function like normalizePath(...) would be beneficial. It would save us from having to write the same replace each time, and can then just use a function where it cares about the specifics internally.

If I think back to how we handled this in [next-on-pages](https://github.com/cloudflare/next-on-pages/blob/96c331d20e679ebad0e43153430786685dbbcdec/packages/next-on-pages/src/utils/fs.ts#L27), we were also accounting for extended-length paths on windows as well as part of that function. Maybe something to consider here too?

Reply:
> I think extracting the (...).replaceAll(..., ...) to a function like normalizePath(...) would be beneficial.

+1 to that

And you can use `path.posix.sep` l21:

```js
.replace(config.paths.standaloneApp.replaceAll(path.sep, path.posix.sep) + path.posix.sep, "")
```

Thanks!

Reply:
> And you can use `path.posix.sep` l21:
> 
> ```js
> .replace(config.paths.standaloneApp.replaceAll(path.sep, path.posix.sep) + path.posix.sep, "")
> ```

Real

Reply:
> I think extracting the (...).replaceAll(..., ...) to a function like normalizePath(...) would be beneficial. It would save us from having to write the same replace each time, and can then just use a function where it cares about the specifics internally.

I think it may not be called `normalizePath` because something like `C:/Windows` does not seem normal on Windows

Reply:
normalize is a common term to refer to the process of converting different inputs to a common format. But let's defer that to a follow up PR and merge this one to unblock normal WIndows users ;)